spec/e2e/accessibility.test.js

```diff
@@ -0,0 +1,66 @@
+import { describe, before, after, test, it } from 'node:test';
+import assert from 'node:assert';
+import { relative, dirname } from 'node:path';
+import glob from 'fast-glob';
+import puppeteer from 'puppeteer';
+import { AxePuppeteer } from '@axe-core/puppeteer';
+import { createServer } from 'http';
+import handler from 'serve-handler';
+import getPort from 'get-port';
+
+const paths = glob
+  .sync('_site/**/index.html')
+  .map((path) => dirname(relative('_site', path)))
+  .filter((path) => path !== '.')
+  .map((path) => `/${path}/`);
+
+describe('accessibility', () => {
+  /** @type {Server} */
+  let server;
+
+  /** @type {number} */
+  let port;
+
+  /** @type {import('puppeteer').Browser} */
+  let browser;
+
+  before(async () => {
+    port = await getPort();
+    browser = await puppeteer.launch();
+    server = createServer((request, response) =>
+      handler(request, response, { public: '_site' }),
+    ).listen(port);
+  });
+
+  after(async () => {
+    await Promise.all([browser.close(), server.close()]);
+  });
+
+  it('has pages to test', () => {
+    assert(paths.length);
+  });
+
+  paths.forEach((path) => {
+    test(path, async () => {
+      const page = await browser.newPage();
+      await page.goto(`http://localhost:${port}${path}`);
+      const results = await new AxePuppeteer(page)
+        .withTags(['wcag2a', 'wcag2aa'])
```

Comment:
Depending how strict you want the checks, there's a number of "best practices" rules that aren't strictly tied to WCAG that you could opt-in to. I suspect it'd catch a handful more issues, but issues still worth taking a look at. Up to y'all 🤷 

Related: https://github.com/18F/identity-design-system/pull/435#discussion_r1532727802

Reply:
i think having stricter checks would be good! we should fix the best practice issues.

Reply:
Thanks for flagging this. I will include it in the follow up ticket so we can add them once we've knocked out some of the higher urgency items.